src/containers/Pomodoro/Pomodoro.js

```diff
@@ -36,17 +42,36 @@ class Pomodoro extends Component {
     const { seconds } = this.state;
     const timeout = setTimeout(() => {
       clearInterval(timer);
-      this.setState({ timerIsRunning: false, seconds: seconds, sessionSoundStatus: Sound.status.PLAYING });
+      if (Notification.permission === 'granted') {
```

Comment:
You probably shouldn't care about this right now (because - f█k old browsers), but if you're wrapping permission request in a check `if ('Notification' in window) ...` you should do it everywhere, just for the sake of consistency, which should generally brings good things 🙂